src/components/Logs/index.tsx

```diff
@@ -7,31 +7,8 @@ type LogsProps = {
 };
 
 export function LogsViewer({ logs }: LogsProps) {
-  const [variant, setVariant] =
+  const [variant] =
```

Comment:
**Something to note:** 

I have come across something like this before, and it's weird because if the variable doesn't change, then it should just be a constant rather than a state variable.

Reply:
sure, it make sense in cases where you want to memoize a value.